TGS3.json

```diff
@@ -1,7 +1,22 @@
 {
     "documentation": "/tg/station server 3 configuration file",
+    "changelog": {
+      "script": "tools/ss13_genchangelog.py",
+      "arguments": "html/changelog.html html/changelogs",
+      "pip_dependancies": [
+        "PyYaml",
+        "beautifulsoup4"
+      ]
+    },
+    "synchronize_paths": [
+      "html/changelog.html",
+      "html/changelogs/*"
+    ],
     "static_directories": [
       "config",
       "data"
+    ],
+    "dlls": [
+      "libmysql.dll"
```

Comment:
Add `rust_g.dll` here also

Reply:
Works fine without it and TG removed it from their json a while back, probably for the best if someone wants to update the dll or stuff, didn’t cause any problems in my tests